configure.ac

```diff
@@ -461,13 +469,25 @@ AC_LINK_IFELSE([AC_LANG_PROGRAM([[
 AC_SUBST(RKT_DEFINES_FOR_ENTER)
 
 # Check whether we can build TPM support code
-AC_CHECK_HEADER(trousers/tss.h, [TPM_TAGS=tpm], [], [AC_INCLUDES_DEFAULT])
+AC_CHECK_HEADER(trousers/tss.h, [HAVE_TPM=yes], [HAVE_TPM=no], [AC_INCLUDES_DEFAULT])
+
+TPM_TAGS=''
+AS_CASE([${RKT_ENABLE_TPM}],
+        [no],
+	        dnl nothing to do
+	        [:],
+	[auto],
+	        [AS_VAR_IF([HAVE_TPM],[yes],
+		           [TPM_TAGS=tpm])],
+	[yes],
+	        [AS_VAR_IF([HAVE_TPM],[no],
+		           [AC_MSG_ERROR([*** TPM is enabled, but could not find required development files])])
+		 TPM_TAGS=tpm],
+	[AC_MSG_ERROR([*** Invalid value passed to --enable-tpm, should be either 'yes', 'no' or 'auto'])])
```

Comment:
Can `./configure` say whether TPM is enabled or not? (`AC_MSG_RESULT` at the end)